src/store/thunks/posts.js

```diff
@@ -1,18 +1,41 @@
 import { postsFetched } from "store/actions/posts";
 import { db } from "database";
 
-export const fetchPosts = () => (dispatch) => {
-  db.collection("posts")
-    .orderBy("created_at", "desc")
-    .get()
-    .then((snapshot) => {
-      const postArray = [];
-      snapshot.forEach((post) => {
-        const postObj = post.data();
-        postObj.id = post.id;
-        postArray.push(postObj);
-      });
-      dispatch(postsFetched(postArray));
-    })
-    .catch(console.error);
+export const fetchPosts = (sortBy, timeframe) => (dispatch) => {
```

Comment:
Can this take an object as argument as per usual? `fetchPosts = ({ sortBy, timeframe })`

Reply:
done